charters/DataBrowser/charter.md

```diff
@@ -0,0 +1,67 @@
+# Data Browser
+
+
+## Description
+The Data Browser (available via the Data Portal 'Explore' section) is an application that provides a faceted search capability allowing users to search and download subsets of cellular resolution data hosted by the Data Coordination Platform (DCP).
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyzes, and make available the data in the Human Cell Atlas project.
+
+*Data Portal* The landing site for the Human Cell Atlas (HCA) DCP.
+
+*Analysis Applications* Third party portals listed in the 'Analyze' section of the Data Portal UI.
+
+## Objectives
+The objective of the Data Browser is to provide an easy-to-use User Interface (UI) allowing users to search, download and analyze data stored in the DCP at petabyte scale on public clouds.
```

Comment:
This suggests just user interface, whereas the core capabilities describe an API too. Update this to resolve the conflict (I think you probably want to add API here?)

Reply:
Agreed.